Solutions/DNS Essentials/NgrokReverseProxyOnNetwork

```diff
@@ -0,0 +1,30 @@
+id: 50b0dfb7-2c94-4eaf-a332-a5936d78c263
```

Comment:
@praveenthepro can you place this file under under following path?
https://github.com/Azure/Azure-Sentinel/tree/1dee9c27f3bc886fe690fc15d49e9b7055b462e7/Solutions/DNS%20Essentials/Analytic%20Rules

Reply:
@praveenthepro can you please validate the rule have been moved under Analytics Rules? I don't see this change

Reply:
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/10a91f06-2693-4c62-a00d-60b10e192e49)

Reply:
Thanks for noticing, Now changed the path, Please checkout